http-server/src/main/java/org/triplea/modules/error/reporting/ErrorReportController.java

```diff
@@ -11,17 +12,38 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.core.Context;
 import lombok.Builder;
+import org.jdbi.v3.core.Jdbi;
+import org.triplea.http.AppConfig;
 import org.triplea.http.HttpController;
 import org.triplea.http.client.error.report.ErrorReportClient;
 import org.triplea.http.client.error.report.ErrorReportRequest;
 import org.triplea.http.client.error.report.ErrorReportResponse;
+import org.triplea.http.client.github.issues.GithubIssueClient;
 
 /** Http controller that binds the error upload endpoint with the error report upload handler. */
 @Builder
 public class ErrorReportController extends HttpController {
   @Nonnull
   private final BiFunction<String, ErrorReportRequest, ErrorReportResponse> errorReportIngestion;
 
+  public static ErrorReportController build(final AppConfig configuration, final Jdbi jdbi) {
+    final GithubIssueClient githubIssueClient =
+        GithubIssueClient.builder()
+            .uri(AppConfig.GITHUB_WEB_SERVICE_API_URL)
+            .authToken(configuration.getGithubApiToken())
+            .githubOrg(AppConfig.GITHUB_ORG)
+            .githubRepo(configuration.getGithubRepo())
+            .build();
+
+    if (githubIssueClient.isTest()) {
```

Comment:
Can you clarify what this block is for?

Reply:
It is moved code. I can't find where @RoiEXLab had a similar question, that answer would have been useful here.

In short it's a validation check of configuration to ensure we do not have an 'issue client' set to a 'test' mode with a prod configuration set. 

The `isTest` check is defined as:

```
  public boolean isTest() {
    return authToken.equalsIgnoreCase("test");
  }
```

In short we get a stubbing behavior when the authtoken used by the server to create github issues is set to 'test'.